notebooks/Specviz2dExample.ipynb

```diff
@@ -61,8 +69,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from astropy.utils.data import download_file\n",
-    "fn = download_file('https://stsci.box.com/shared/static/exnkul627fcuhy5akf2gswytud5tazmw.fits', cache=True)"
+    "fn = \"jw01538-o160_s00004_nirspec_f170lp-g235h-s1600a1-sub2048_s2d.fits\"\n",
+    "uri = f\"mast:JWST/product/{}\"\n",
```

Comment:
```suggestion
    "uri = f\"mast:JWST/product/{fn}\"\n",
```

Reply:
How did I mess these up between testing and my last commit?? Ugh, thanks.